server/api/crud/alert_template.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sqlalchemy.orm
+
+import mlrun.utils.singleton
+import server.api.api.utils
+import server.api.utils.helpers
+import server.api.utils.singletons.db
+
+
+class AlertTemplate(
```

Comment:
```suggestion
class AlertTemplates(
```

CRUDs are named in plural when dealing with a specific object-type. Exceptions are usually when we deal with functionality (such as `FeatureStore`) that has multiple objects.